Sources/Packages.php

```diff
@@ -2644,6 +2644,9 @@ function build_special_files__recursive($path, &$data)
 			$dont_chmod = false;
 			while ($entry = readdir($dh))
 			{
+				if ($entry == '.' || $entry == '..')
+					continue;
```

Comment:
```suggestion
				// Bypass directory abbreviations altogether...
				if ($entry == '.' || $entry == '..')
					continue;
```

It works, but you forgot to comment it.

Reply:
rather than argue with you, I'm just going to commit it

Reply:
I wish you good luck and all the best.